### PR TITLE
[JB][OPS-10958] configure childbenefitsrepayments notifications not t…

### PR DESCRIPTION
**Pull request by barnesjake**
…o be sent to des

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -150,6 +139,9 @@ mibops.poller.initialDelay = 14 seconds
 mibops.poller.interval = 7 seconds
 mibops.poller.pollLimit = 200
 
+# todo look into removing this, it makes no sense and is only useful in the itTests
```
Comment (barnesjake) [inline conf/application.conf:142]: Raised a ticket for this https://jira.tools.tax.service.gov.uk/browse/OPS-11289

Comment (pawelpanasewicz) [reply conf/application.conf:142]: could you add ticket number to the comments please

Comment (barnesjake) [reply conf/application.conf:142]: sure thing

Comment (pawelpanasewicz) [reply conf/application.conf:142]: or simply remove it, probably faster then all time spent on managing this ticket.

**Review by pawelpanasewicz — APPROVED** [review]

File: conf/application.conf (hunk 1 above)
Comment (barnesjake) [reply conf/application.conf]: I did toy with the idea, but it's a lot more work (in the tests)

**Review by pawelpanasewicz — APPROVED** [review]

**Pull request merged**